### ISSUE TITLE
Space in volume name causes these errors ...

### ISSUE THREAD

**Issue by mhulse** [rvm/rvm#2205]
Hi,

I'm one of those dummkopfs with a space in thier home directory path.

In this case, I have a space in my home directory volume name (I regret the day I ever decided that was a good idea).

Prompted by an initial install of `rvm`, I followed these instructions:

``` bash
sudo mkdir -p /mhulse.rvm
sudo chown -R "mhulse:" /mhulse.rvm
echo "export rvm_path=/mhulse.rvm" >> "/Volumes/SCRATCH 1/mhulse/.rvmrc"
```

Now, when running `$ \curl -L https://get.rvm.io | bash -s stable`, I get this after installation:

``` bash
Installing RVM to /mhulse.rvm/
    Adding rvm PATH line to /Volumes/SCRATCH 1/mhulse/.bashrc /Volumes/SCRATCH 1/mhulse/.zshrc.
touch: 1/mhulse/.bashrc: No such file or directory
/mhulse.rvm/src/rvm/scripts/functions/installer: line 1051: 1/mhulse/.bashrc: No such file or directory
touch: 1/mhulse/.zshrc: No such file or directory
/mhulse.rvm/src/rvm/scripts/functions/installer: line 1051: 1/mhulse/.zshrc: No such file or directory
    Adding rvm loading line to /Volumes/SCRATCH 1/mhulse/.bash_profile /Volumes/SCRATCH 1/mhulse/.zprofile.
touch: 1/mhulse/.bash_profile: No such file or directory
/mhulse.rvm/src/rvm/scripts/functions/installer: line 1083: 1/mhulse/.bash_profile: No such file or directory
touch: 1/mhulse/.zprofile: No such file or directory
/mhulse.rvm/src/rvm/scripts/functions/installer: line 1083: 1/mhulse/.zprofile: No such file or directory
```

...  and running `$ rvm get stable`, this shows after updating:

``` bash
Upgrading the RVM installation in /mhulse.rvm/
    RVM PATH line found in /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH.
    RVM PATH line not found for Bash or Zsh, run the installer with '--auto-dotfiles' to fix it.
    RVM sourcing line found in /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH /Volumes/SCRATCH.
    RVM sourcing line not found for Bash or Zsh, run the installer with '--auto-dotfiles' to fix it.
Upgrade of RVM in /mhulse.rvm/ is complete.
```

Basically, I have the same/similar problem as #501 and #1760 ...

Here's what version my system says I've got installed:

```
$ rvm -v
rvm 1.22.7 (stable) ...
```

I'm also using Homebrew.

Just curious if there's anything I can do to fix, or should I go about the process of removing spaces in my volume names (this isn't the first time I've run into this type of problem)?

Thanks so much! Sorry to be a bother.
